### PR TITLE
Upgrade/rails 6 rebased

### PR DESCRIPTION
**Pull request by yachtcaptain23**

### CODE REVIEW

**Comment by corymcdonald** [comment]
Closes #2185

File: Gemfile [hunk 1]
```diff
@@ -16,14 +16,11 @@ gem 'activerecord-session_store'
...
 
-# Pagination
-gem "api-pagination"
-
 # Encrypt DB data at rest
 gem "attr_encrypted", "~> 3.1.0"
 
 # Integration with Matomo Piwik
-gem 'autometal-piwik', :require => 'piwik', git: "https://github.com/matomo-org/piwik-ruby-api.git", branch: "master"
+gem 'autometal-piwik', :require => 'piwik', git: "https://github.com/yachtcaptain23/piwik-ruby-api.git", branch: "feature/rails_6_support"
```
Comment (diracdeltas) [inline Gemfile]: any reason not to publish this under brave on github?

Comment (yachtcaptain23) [reply Gemfile]: That I can do. Will drive this.

**Comment by diracdeltas** [comment]
lgtm. for future reference, https://github.com/brave-intl/publishers/pull/2287/files#diff-8b7db4d5cc4b8f6dc8feb7030baa2478R76 should probably also be moved to a brave repo

**Pull request merged**